Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -46,6 +46,7 @@
     custom_suffix = '-uat'
 
 BUILD_WEBKIT_HOSTNAMES = ['build.webkit.org', 'build']
+TESTING_ENVIRONMENT_HOSTNAMES = ['build.webkit-uat.org', 'build-uat']
```

Comment:
we should also add build-dev in this list (similar thing for ews). cc @lingcherd

Reply:
I see TESTING_ENVIRONMENT_HOSTNAMES used only in the conditions on line 1389 and 1450 where BUILD_WEBKIT_HOSTNAMES is also tested. So is this list variable even necessary? In all other cases, you are testing if the host is in BUILD_WEBKIT_HOSTNAMES.

Reply:
Then are you suggesting that we add these hosts to BUILD_WEBKIT_HOSTNAMES? Or not have them checked in at all, and you have to manually add them while testing.